rpc/js/package.json

```diff
@@ -23,18 +23,22 @@
     "ts-protoc-gen": "^0.14.0",
     "tslib": "^2.4.1",
     "typescript": "^5.2.2",
-    "vite": "^3.1.4",
-    "webpack": "^5.74.0",
-    "webpack-cli": "^4.10.0"
+    "vite": "^5.0.6"
```

Comment:
We haven't yet migrated other projects to Vite v5. If we don't have any issues here, great! But it may be prudent to stick to Vite v4 if we run into weird stuff, especially given how much annoying CJS stuff we have to deal with in this project